ClientApp/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,4 @@
 export const environment = {
    production: true,
+   VERSION: require('../../package.json').version,
```

Comment:
Some explanation on how to use version information might be useful at the next meeting, I'm curious.